spec/acceptance/sqlserver_database_spec.rb

```diff
@@ -298,60 +290,19 @@ def run_sql_query_opts(query, expected_row_count)
             require => Sqlserver::Database['#{@db_name}'],
         }
       MANIFEST
-      ensure_sqlserver_database(pp)
+      apply_manifest(pp, catch_failures: true)
 
       puts 'Validate that a table can be created in the database:'
       query = "USE #{@db_name}; SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE table_name = '#{@table_name}';"
-      run_sql_query(host, run_sql_query_opts(query, 1))
+      run_sql_query(run_sql_query_opts(query, 1))
 
       puts "validate the Database '#{@db_name}' has correct default_language setting:"
       query = "SELECT name AS Database_Name, default_language_name
                 FROM sys.databases
                 WHERE name = '#{@db_name}'
                 AND default_language_lcid = '1028';"
 
-      run_sql_query(host, run_sql_query_opts(query, 1))
-      # rubocop:enable RSpec/InstanceVariable
-    end
-
-    it 'Test Case MODULES-10335: Create two database with seperate sp_configure', tier_low: true do
```

Comment:
Just curious as to why we removed this test case?

Reply:
Converting to litmus unearthed a hidden warning that I could not get around.